nnstreamer_example/example_object_detection/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# demo app
+pkg_check_modules(cairo_pkg REQUIRED cairo)
```

Comment:
Don't we have to add cairo package into 'debian/control' build-depends?

Reply:
https://github.com/nnsuite/nnstreamer/blob/f62e5c5f71c445049d37114f3625a13576bb4f18/debian/control#L9

it's updated. thanks :)